.github/workflows/release.yaml

```diff
@@ -264,7 +264,11 @@ jobs:
       - uses: actions/checkout@v3
       - name: Create tutorials tar
         run: |
-          tar -cvf mlrun-tutorials.tar docs/tutorials
+          mkdir jupyter-welcome
+          wget -c https://github.com/v3io/tutorials/blob/development/welcome.ipynb -o jupyter-welcome/welcome.ipynb
```

Comment:
Pulling from development is dangerous. Someone can easily change the file names or whatever and break release without knowing. Let's create a new branch in `v3io/tutorials` and use that instead. Call it something like `mlrun-release-stable`.

Reply:
Alright no sweat, will wait @aviaIguazio to do that since i don't have permissions

Reply:
@alonmr Will change branch soon

Reply:
Changed branch to mlrun-release-stable